modulefiles/build.hera.intel

```diff
@@ -23,7 +25,11 @@ module load nemsiogfs/2.5.3
 module load landsfcutil/2.4.1
 module load wgrib2/2.0.8
 
-module use /scratch1/NCEPDEV/nems/emc.nemspara/soft/modulefiles
-module load hdf5_parallel/1.10.6.release
-module load netcdf_parallel/4.7.4.release
-module load esmf/8.0.0_ParallelNetCDF
+module load jasper/2.0.15
+module load zlib/1.2.11
+module load png/1.6.35
+
+module load hdf5/1.10.6
+module load netcdf/4.7.4
+module load nccmp/1.8.7.0
+module load esmf/8_1_0_beta_snapshot_21
```

Comment:
Does Hera.Intel not have beta snapshot 27?

Reply:
There are two versions of snapshot 27 - a 'debug' version and a 'normal' version.

Reply:
I can test with '27'.

Reply:
I say that because UFS modules load `27`.
We should figure out a way to load the same modules as UFS (this is UFS_utils afterall).

Reply:
Done at 276fd59.